src_c/scrap.c

```diff
@@ -355,6 +408,77 @@ _scrap_set_mode(PyObject *self, PyObject *args)
 
 #endif /* !defined(MAC_SCRAP) */
 
+/**
+ * @brief Fetches a python string from the SDL clipboard. If
+ *        there is nothing in the clipboard, it will return empty
+ *
+ * @return PyObject*
+ */
```

Comment:
For me this method raises a pygame.error with no message if the clipboard is empty.

Reply:
Fixed this by checking if `err` is empty